iTunesSearchHW23/iTunesSearch/Flows/StoreItemCollectionViewController.swift

```diff
@@ -0,0 +1,65 @@
+// MARK: - Importing frameworks
+import UIKit
+
+// MARK: - Classes
+class StoreItemCollectionViewController: UICollectionViewController {
+    //MARK: - Life Cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        collectionView.register(StoreItemCollectionViewSectionHeader.self, forSupplementaryViewOfKind: "Header", withReuseIdentifier: StoreItemCollectionViewSectionHeader.reuseIdentifier)
+    }
+    
+    // MARK: - Methods
+    func configureCollectionViewLayout(for searchScope: SearchScope) {
+        let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1/3), heightDimension: .fractionalHeight(1.0))
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+        item.contentInsets = .init(top: 8, leading: 5, bottom: 8, trailing: 5)
+        
+        let groupSize = NSCollectionLayoutSize(widthDimension: searchScope.groupWidthDimension, heightDimension: .absolute(166))
+        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitem: item, count: searchScope.groupItemCount)
+        
+        let section = NSCollectionLayoutSection(group: group)
+        section.orthogonalScrollingBehavior = searchScope.orthogonalScrollingBehavior
+        
+        let headerSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .absolute(28))
+        let headerItem = NSCollectionLayoutBoundarySupplementaryItem(layoutSize: headerSize, elementKind: "Header", alignment: .topLeading)
+        
+        section.boundarySupplementaryItems = [headerItem]
+        
+        let layout = UICollectionViewCompositionalLayout(section: section)
+        collectionView.collectionViewLayout = layout
+    }
+}
+
+// MARK: - Extensions
```

Comment:
If u add such mark, it should contain name of object or short description of functionality